src/fmu/sumo/uploader/_connection.py

```diff
@@ -43,4 +34,4 @@ def refresh(self):
 
     def _establish_connection(self):
         """Establish the connection with Sumo API, take user through 2FA."""
-        return CallSumoApi(env=self.env)
+        return SumoClient(env=self.env)
```

Comment:
`SumoClient` won't prompt for user login until a request has been made and we detect no valid token in the cache, so technically the `_establish_connection` won't establish the connection. Not sure if it matters.

Reply:
The logic was left unchanged in this PR. Maybe we can keep it as-is until next iteration (if necessary).